src/lua/init.lua

```diff
@@ -407,7 +420,13 @@ return function(args)
     if not target or #target == 0 then return commonBundle(bundle) end
     return buildBundle(target, bundle)
   end
-  local usage = [[Luvi Usage Instructions:
+  local prefix = [[$(LUVI) $(LUVI_VERSION)]]
+  prefix = string.gsub(prefix, "%$%(LUVI%)", args[0])
+  prefix = string.gsub(prefix, "%$%(LUVI_VERSION%)", luvi.version)
```

Comment:
Lua has string.format that works a lot like sprintf.  You should probably use that instead of gsubs.